nuxt.config.ts

```diff
@@ -20,6 +20,9 @@ export default defineNuxtConfig({
     apiHost: 'https://plausible.dev.service.aepps.com',
     trackLocalhost: false,
   },
+  appConfig: {
+    APP_VERSION: process.env.APP_VERSION || 'UNKNOWN',
```

Comment:
Can't we simply hide that if it's empty? So the `|| ' UNKNOWN'` is not needed

Reply:
Great idea, didn't think of that :sweat_smile: